app/views/reviews/new.html.erb

```diff
@@ -20,6 +20,6 @@
   <%= f.label :body %>
   <%= f.text_area :body %>
 
-  <%= f.submit "Add Review" %>
+  <%= f.submit "Add Review", :class => "button radius"  %>
```

Comment:
The hashrocket syntax is outdated! Instead, it should look like: `class: "button radius"`